src/libkomwm.py

```diff
@@ -640,9 +640,9 @@ def rule_sort_key(dict_):
                     st = dict([(k, v) for k, v in st.items() if str(v).strip(" 0.")])
                     if 'width' in st or 'pattern-image' in st:
                         has_lines = True
-                    if 'icon-image' in st or 'symbol-shape' in st or 'symbol-image' in st:
+                    if 'icon-image' in st and st.get('icon-image') != 'none' or 'symbol-shape' in st or 'symbol-image' in st:
```

Comment:
Hm, just noticed, what do symbol-shape and symbol-image mean? Are they obsolete?

Reply:
`symbol-shape` are for "circles", I don't know if they work, a quick experiment didn't show anything

`symbol-image` is used in one line only :) so redundant for sure